packages/beachball/src/git/index.ts

```diff
@@ -81,25 +87,34 @@ export function fetchRemote(remote: string, remoteBranch: string, cwd: string) {
 
 export function getChanges(branch: string, cwd: string) {
   try {
-    const results = git(['--no-pager', 'diff', '--name-only', branch + '...'], { cwd });
-
-    if (!results.success) {
-      return [];
-    }
-
-    let changes = results.stdout;
-
-    let lines = changes.split(/\n/) || [];
+    return processGitOutput(git(['--no-pager', 'diff', '--name-only', branch + '...'], { cwd }));
+  } catch (e) {
+    console.error('Cannot gather information about changes: ', e.message);
+  }
+}
 
-    return lines
-      .filter(line => line.trim() !== '')
-      .map(line => line.trim())
-      .filter(line => !line.includes('node_modules'));
+export function getChangesBetweenRefs(fromRef: string, toRef: string, options: string[], cwd: string) {
+  try {
+    return processGitOutput(git(['--no-pager', 'diff', '--name-only', ...options, fromRef, toRef, '.'], { cwd }));
   } catch (e) {
     console.error('Cannot gather information about changes: ', e.message);
   }
 }
 
+function processGitOutput(output: ProcessOutput) {
+  if (!output.success) {
+    return [];
+  }
+
+  let changes = output.stdout;
```

Comment:
nit: I think 'change' should be something else here

Reply:
Good catch! I've also noticed that `getStagedChanges` could benefit from using `processGitOutput` and updated it :)